### ISSUE TITLE
Feat: Editing the Guidelines Page

### ISSUE THREAD

**Issue by Hemashree21** [Nactore-Org/Nacto-Care#160]
### Describe the feature

Editing the guidelines page to add the correct information relating to our project.
@Harshil-Jani can you please assign this?

### Add ScreenShots

![image](https://github.com/Nactore-Org/Nacto-Care/assets/138188163/d3347d66-3ae7-4cfe-9148-c88886898201)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by Harshil-Jani** [comment]
@Hemashree21 I think we should not edit the guidelines page (If the colours and fonts mentioned are good enough). Instead based on the guidelines page we need to check in whole website where we are not following it. For example the green of Login and Signup is different from the green of "Search" in below SS.  Now since we have guidelines to follow, I believe we need to make sure that we are following it. Also if something is more prevelant we can add it up to guidelines like how the button border should be etc. 

![image](https://github.com/Nactore-Org/Nacto-Care/assets/79367883/bb9779cd-3303-4b38-879a-cb57c59867f9)

Also I believe this should be a low priority issue right now since the site is changing everyday and If we put strict guidelines it would not be so flexible for contributors to contribute since people tend to skip "CONTRIBUTING or Branding Guidelines" etc. Maybe we can cover this up later ?

If you can find some other issues to work on let's keep this for later or else you can start now too but I believe it will be more maintainance specific and would require frequent iterations over a period of time.

**Comment by Hemashree21** [comment]
Yeah sure, I can work on it once the majority of the project is complete. Right now I am working on the FAQ section and once I finish that, I will look for other issues.
